R/z_print.R

```diff
@@ -83,6 +83,11 @@ print.animint <- function(x, ...){
 animint <- function(...){
   L <- list(...)
   
+  # Check if argument list is empty
+  if(length(L) == 0) {
+    stop(paste("No arguments passed to animint"))
```

Comment:
Could you please make the error message more informative? Please tell the user how to fix the problem.
something like "No arguments passed to animint, but arguments should be ggplots (1 or more) and options (0 or more)"
Also please remove paste as it is not necessary.